hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/RequestConverter.java

```diff
@@ -579,9 +579,9 @@ public static BulkLoadHFileRequest buildBulkLoadHFileRequest(
    * @return a bulk load request
    */
   public static BulkLoadHFileRequest buildBulkLoadHFileRequest(
-      final List<Pair<byte[], String>> familyPaths,
-      final byte[] regionName, boolean assignSeqNum,
-      final Token<?> userToken, final String bulkToken, boolean copyFiles) {
+      final List<Pair<byte[], String>> familyPaths, final byte[] regionName, boolean assignSeqNum,
+        final Token<?> userToken, final String bulkToken, boolean copyFiles,
+          List<String> clusterIds) {
```

Comment:
Looks like we use `List<UUID>` in the WALEntry code path. Is there a reason to use `List<String>`?

Reply:
Oh, you did String because of protobuf? It might be nice to always refer to clusterId's via UUID and just hide the fact that we serialize them as normal strings in protobuf (behind ProtobufUtil API).

Reply:
I agree keeping UUIDs until serialization would be nicer.

Reply:
That's great.